onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -30,6 +30,11 @@ enum class WebnnDeviceType {
   GPU,
 };
 
+typedef struct {
+  std::string opName;
+  bool isCpuSupported;
```

Comment:
Ah, so this really means *XNNPack* supports it, not the CPU EP - I was confused for a bit :D. Hmm, could we add a comment? I wager this will be a temporary field anyway, once XNNPack catches up to the rest of the ML backends.

e.g. `bool isCpuSupported;  // The WebNN CPU backend XNNPack supports it (not about the CPU EP).`

Reply:
Thank u! Done.